.pre-commit-config.yaml

```diff
@@ -13,3 +13,16 @@ repos:
         additional_dependencies:
           - "types-pytz"
           - "types-requests"
+
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args:
+          - "--line-length=99"
```

Comment:
enforcing a specific line length can hinder readability - line length limits don't provide any benefits IMO.
Let's set to `120` since that's still reasonable.

FWIW here's existing max line lengths.

```
(.venv) hashhar@Ashhars-MacBook-Pro trino-python-client % git ls-files | xargs -n1 gwc -L | sort -n
0 tests/__init__.py
0 tests/integration/__init__.py
0 tests/unit/__init__.py
0 tests/unit/sqlalchemy/__init__.py
30 tox.ini
54 .pre-commit-config.yaml
74 tests/unit/test_transaction.py
74 trino/constants.py
74 trino/logging.py
74 trino/sqlalchemy/__init__.py
75 trino/__init__.py
77 LICENSE
79 tests/unit/test_http.py
80 tests/unit/sqlalchemy/conftest.py
81 trino/exceptions.py
87 tests/integration/conftest.py
87 tests/unit/sqlalchemy/test_datatype_split.py
87 trino/transaction.py
95 .gitignore
95 trino/sqlalchemy/compiler.py
96 tests/unit/oauth_test_utils.py
97 setup.cfg
98 setup.py
101 trino/sqlalchemy/datatype.py
102 tests/unit/sqlalchemy/test_datatype_parse.py
106 tests/unit/sqlalchemy/test_compiler.py
109 tests/integration/test_dbapi_integration.py
109 tests/unit/sqlalchemy/test_dialect.py
110 tests/integration/test_sqlalchemy_integration.py
116 trino/sqlalchemy/error.py
117 trino/auth.py
117 trino/client.py
118 tests/unit/test_dbapi.py
118 trino/sqlalchemy/dialect.py
120 trino/dbapi.py
131 .github/workflows/ci.yml
184 tests/unit/conftest.py
192 tests/unit/test_client.py
438 README.md
```